README.md

```diff
@@ -151,6 +151,31 @@ func main() {
 }
 ```
 
+### Sending Metrics
```

Comment:
Maybe rename this to _Flushing Metrics_?

Reply:
Yeah you're totally right.